list.go

```diff
@@ -153,6 +154,10 @@ func (db *RoseDB) LIndex(key []byte, index int) ([]byte, error) {
 		return nil, err
 	}
 
+	if seq >= tailSeq || seq <= headSeq {
+		return nil, ErrWrongIndex
```

Comment:
the same as line 188?

Reply:
IMO, if the index location is not legal after the logical index has been converted to a physical index, then an error should be returned. Does this code cause any problems? I'm not completely familiar with the code of the project, so I would appreciate any feedback I can get!